pkg/app/couchbase.go

```diff
@@ -106,7 +107,10 @@ func (cb *CouchbaseDB) Install(ctx context.Context, ns string) error {
 
 	// Install cb cluster
 	err = cli.Install(ctx, fmt.Sprintf("%s/%s", cb.operatorChart.RepoName, cb.clusterChart.Chart), cb.clusterChart.Version, cb.clusterChart.Release, cb.namespace, cb.clusterChart.Values)
-	return errors.Wrapf(err, "Failed to install helm chart. app=%s chart=%s release=%s", cb.name, cb.clusterChart.Chart, cb.clusterChart.Release)
+	if err != nil {
```

Comment:
if err is nil, errors.Wrapf would return nil automatically right. Or thats not happening.

Reply:
Okay, wasn't aware of that. Will revert this.